generators/app/templates/javascript/_package.json

```diff
@@ -21,22 +23,27 @@
     "echo",
     "amazon"
   ],
-  "license":"ISC",
+  "license": "ISC",
   "dependencies": {
-    "bluebird":"^3.3.5",
-    "lodash":"^4.17.11",
-    "voxa":"^3.0.0-alpha32"
+    "bluebird": "^3.3.5",
+    "express": "^4.16.4",
+    "lodash": "^4.17.11",
+    "morgan": "^1.9.1",
+    "raven": "^2.6.4",
+    "serverless-sentry": "^1.2.0",
+    "serverless-sentry-lib": "^1.1.2",
+    "voxa": "3.0.0-alpha39"
```

Comment:
Can you make sure all these dependencies are up to date?

Reply:
Ok